src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -33,26 +33,27 @@ cursor until the `Batch` ID changes. Whereupon, the consolidator will find the
 next cursor with the next lowest batch ID (which should be greater, of course,
 than the `Batch` value we were just iterating on).
 
-Put another way: if we called `GetRowCursor` (possibly with an `IRandom`
+Put another way: suppose we called `GetRowCursor` (possibly with a `Random`
 instance), and we store all the values from the rows from that cursoring in
 some list, in order. Now, imagine we create `GetRowCursorSet` (with an
-identically constructed `IRandom` instance), and store the values from the
-rows from the cursorings from all of them in a different list, in order,
+identically constructed `Random` instance), and store the values from the rows
+from the cursorings from all of them in a different list, in order,
 accompanied by their `Batch` value. Then: if we were to perform a *stable*
 sort on the second list keyed by the stored `Batch` value, it should have
 content identical to the first list.
 
-So: `Batch` is a `long` value associated with every `ICounted` implementation
-(including implementations of `ICursor`). This quantity must be:
-
-Non-decreasing as we call `MoveNext` or `MoveMany`. That is, it is fine for
-the `Batch` to repeat the same batch value within the same cursor (though not
-across cursors from the same set), but any change in the value must be an
-increase.
+So: `Batch` is a `long` value associated with every `DataViewRow` instance.
+This quantity must be non-decreasing as we call `MoveNext` or `MoveMany`. That
```

Comment:
Ahhh. I just noticed I forgot to remove references to `MoveMany` which was removed as part of #2195.